### PR TITLE
Remove `suffix` parameter from scripts

### PR DESCRIPTION
**Pull request by JackPGreen**
`suffix` is a hardcoded derivative of `variant`, and can instead be derived dynamically.

`tag_image_push_rhel.yml` used `suffix`, but never actually set it (unlike `tag_image_push.yml` which did), so hardcoded it to `""` instead.

Fixes: https://github.com/hazelcast/hazelcast-docker/issues/753

### CODE REVIEW

**Review by ldziedziul — COMMENTED** [review]
Am I missing something or `matrix.suffix` is not defined and always empty in `tag_image_push_rhel.yml`?

**Comment by JackPGreen** [comment]
> Am I missing something or `matrix.suffix` is not defined and always empty in `tag_image_push_rhel.yml`?

Glad it's not just me.

**Comment by JackPGreen** [comment]
> > Am I missing something or `matrix.suffix` is not defined and always empty in `tag_image_push_rhel.yml`?
> 
> Glad it's not just me.

Yes, [it's empty](https://github.com/hazelcast/hazelcast-docker/actions/runs/9502129469/job/26189379828):
`TAGS_TO_PUSH=$(get_tags_to_push 5.4.0 "" "17" "$DEFAULT_JDK")`

File: .github/workflows/tag_image_push_rhel.yml [hunk 1]
```diff
@@ -42,6 +42,10 @@ jobs:
       fail-fast: false
       matrix:
         jdk: ${{ fromJSON(needs.jdks.outputs.jdks) }}
+        variant: [ 'slim','' ]
```
Comment (ldziedziul) [inline .github/workflows/tag_image_push_rhel.yml:45]: Why have you added it?

We haven't push slim versions to OpenShift Catalog

Comment (JackPGreen) [reply .github/workflows/tag_image_push_rhel.yml:45]: I _assumed_ it was an oversight as it’s in the non-RHEL job. If that’s wrong I can take it out instead and update the params to suit?

Comment (ldziedziul) [reply .github/workflows/tag_image_push_rhel.yml:45]: There's no request for it so I would keep it as it is (i.e. no slim version on RHEL)

Comment (JackPGreen) [reply .github/workflows/tag_image_push_rhel.yml]: bff66f46b7260e7f3b25124010093e23ec1eec79

**Review by ldziedziul — APPROVED** [review]

**Pull request merged**